PerformanceTests/Sources/PerformanceTestRunner/Core/Util/OperatingSystem.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+enum OperatingSystem: String {
+    case macOS = "macOS"
+    case Linux = "Linux"
+    case Unknown = "Unknown"
```

Comment:
nit: cases of a Swift enum should start with lower case (`macOS`, `linux`, `unknown`)